contracts/utils/TimeLock.sol

```diff
@@ -12,6 +12,7 @@ import { ITimeLock } from '../interfaces/ITimeLock.sol';
 contract TimeLock is ITimeLock {
     bytes32 internal constant PREFIX_TIME_LOCK = keccak256('time-lock');
 
+    // slither-disable-next-line naming-convention
```

Comment:
what's the issue here? can we update the name?

Reply:
yeah it want immutable to this in `THIS_CASE` or `thisCase`.

Reply:
fixed